### DISCUSSION TITLE
There are too many Docker containers, can we consolidate them?

### DISCUSSION THREAD

**Discussion by mapleshadow** [zoriya/Kyoo#476] [General]
### Feature description

There are too many Docker containers, can we consolidate them? It's inconvenient for deployment and management this way.

**Comment by zoriya** [comment]
This was already discussed in #420 but I don't really understand the inconvenience of having many containers.

As a maintainer, having small containers makes features more contained and easier to debug/iterate on. It's also easier if someone wants to contribute because they have less code to worry about.

As a user, you just have to use the provided docker-compose (or the soon to be helm chart, I promise I'll try to finish it soon). Do you find it an inconvenience because you want to run Kyoo via another means?

**Comment by acelinkio** [comment]
v5 removed the need for RabbitMQ, Meiliseach, and Kyoo_Autosync.  So there are a lot less moving parts overall. 

Check out diagram.md for more info on how things interconnect.
